src/ralph/networks/models/networks.py

```diff
@@ -386,7 +414,6 @@ class IPAddress(
     is_gateway = models.BooleanField(
         verbose_name=_('Is gateway address'),
         default=False,
-        editable=False,
```

Comment:
Because we want control this value (similar to is_management) - but I don't see any validation like one gateway (management IP) for whole network.